web/src/api/MercuryAPI.tsx

```diff
@@ -1,9 +1,10 @@
-import { Configuration, TodosApi } from './generated'
+import { ComponentApi, Configuration } from './generated'
 
-export class MercuryAPI extends TodosApi {
+// TODO: Combine all generated api's into MercuryAPI
```

Comment:
Do we have an issue for this? I don't understand.

Reply:
Because of the tags we use on the backend, the generated API is split into 4 different ones.
```
// from web/src/api/generated/api.ts
export * from './api/component-api';
export * from './api/health-check-api';
export * from './api/multiflash-api';
export * from './api/whoami-api';
```

This will lead to duplicated api wrappers like "MercuryAPI" if not combined in some way